bedrock/privacy/templates/privacy/base-protocol.html

```diff
@@ -31,6 +31,7 @@
 {% set navigation_bar = [
   (url('privacy'), 'privacy-home', ftl('privacy-index-mozilla-privacy')),
   (url('privacy.notices.websites'), 'privacy-websites', ftl('privacy-index-mozilla-websites-communications')),
+  (url('privacy.notices.mozilla-accounts'), 'mozilla-accounts', ftl('privacy-index-mozilla-accounts')),
```

Comment:
Got word from Jess that this should use title case to match the rest of the entries.

Reply:
Ok, I'm tempted not to localize this string then I think, since the brand name is `Mozilla accounts` and this may cause some confusion both with the L10n team, and with localizers?

Reply:
Ended up going with an L10n comment to try and explain things and not using a brand name variable. Hopefully that should explain things.

Reply:
Cool. Thanks! I was also wondering if there's a CSS way to make everything there title case?

Reply:
Yeah, we could `text-transform: capitalize` but maybe that's not as reliable as just doing it for real?

Reply:
Yeah, that could work but I think it's perhaps best to let locales deal with how to spell things out directly.

Reply:
Good point 👍